build.sbt

```diff
@@ -24,13 +24,16 @@ lazy val root = (project in file(".")).enablePlugins(
 scalaVersion := "2.11.6"
 
 libraryDependencies ++= Seq(
+  "com.gu" %% "membership-common" % "0.66",
   cache,
   ws,
   PlayImport.specs2,
   "com.gu" %% "play-googleauth" % "0.3.0",
+  "com.gu.identity" % "identity-play-auth_2.11" % "0.5",
```

Comment:
We can sue the double-percent operator, so: `"com.gu.identity" %% "identity-play-auth" % "0.5"`